contracts/libraries/RFQTx.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { RFQOffer, getRFQOfferHash, RFQ_OFFER_TYPESTRING } from "./RFQOffer.sol";
+
+string constant RFQ_TX_TYPESTRING = string(
+    abi.encodePacked("RFQTx(RFQOffer rfqOffer,bytes32 offerHash,address recipient,uint256 takerRequestAmount,uint256 feeFactor)", RFQ_OFFER_TYPESTRING)
```

Comment:
Do we need both `rfqOffer` and `offerHash`  here?

Reply:
The `offerHash` should be removed. f6f45a6